addon.json

```diff
@@ -0,0 +1,74 @@
+{
+    "identifier": "com.seeq.addons.plot_curve",
+    "name": "Plot Curve",
+    "description": "A tool for fitting curves to tabular data and pushing resulting formulas to Seeq",
+    "version": "1.0.0",
```

Comment:
Version `1.0.0` is already in the gallery. You'll need to rev the version number so Add-on Manager can know to display that an update is available.

Reply:
Yeah I think we'll handle that one in a separate PR. I want to keep concrete versioning commits if possible.

Reply:
This is generally updated to work from now on starting with the current version of plot curve: 0.1.3